bin/version-manager.js

```diff
@@ -61,9 +61,6 @@ function buildGlobs(cb) {
   testGlobs.forEach((file) => {
     if (/(?:package\.json|\.js)$/.test(file)) {
       globs.push(file)
-      if (/\.js$/.test(file)) {
```

Comment:
By removing this I can no longer do `npx versioned-tests path/to/test-file.tap.js` which will run one test file against the versioned matrix.  Personally the solution i've done to work around all of this was https://github.com/newrelic/node-test-utilities/pull/104#issuecomment-984707296

Reply:
Top is output without this fix, bottom is with. you can see it runs 10 files vs 1.
```sh
❯ npx versioned-tests --major -i 2 --all test/versioned/hapi/hapi-post-18/plugins.tap.js
Finding tests in 1 globs
Finding all versions for a package
vision(5)
ejs(4)
@hapi/hapi(4)
No version match found. Attempting direct install of ejs@2.5.5
===============================================================
 ✓ hapi-post-18 (1) 1s
===============================================================
PASS

node-newrelic on  main [📦] is 🎁 v8.6.0 via 🤖 v16.13.0 took 3s
❯ npx versioned-tests --major -i 2 --all test/versioned/hapi/hapi-post-18/plugins.tap.js
Finding tests in 1 globs
Finding all versions for a package
ejs(4)
vision(5)
@hapi/hapi(4)
No version match found. Attempting direct install of ejs@2.5.5
===============================================================
 • hapi-post-18/ignoring.tap.js run 5 of 10): success
===============================================================
```